bitnami/postgresql/values.yaml

```diff
@@ -194,29 +194,37 @@ audit:
   logTimezone: ""
 ## LDAP configuration
 ## @param ldap.enabled Enable LDAP support
+## @param ldap.uri LDAP URL beginning in the form `ldap[s]://host[:port]/basedn`
+## DEPRECATED ldap.url It will removed in a future, please use 'ldap.uri' instead
 ## @param ldap.server IP address or name of the LDAP server.
 ## @param ldap.port Port number on the LDAP server to connect to
 ## @param ldap.prefix String to prepend to the user name when forming the DN to bind
 ## @param ldap.suffix String to append to the user name when forming the DN to bind
-## @param ldap.baseDN Root DN to begin the search for the user in
-## @param ldap.bindDN DN of user to bind to LDAP
-## @param ldap.bind_password Password for the user to bind to LDAP
-## @param ldap.search_attr Attribute to match against the user name in the search
-## @param ldap.search_filter The search filter to use when doing search+bind authentication
+## DEPRECATED ldap.baseDN It will removed in a future, please use 'ldap.basedn' instead
+## DEPRECATED ldap.bindDN It will removed in a future, please use 'ldap.binddn' instead
+## DEPRECATED ldap.bind_password It will removed in a future, please use 'ldap.bindpw' instead
+## @param ldap.basedn Root DN to begin the search for the user in
+## @param ldap.binddn DN of user to bind to LDAP
+## @param ldap.bindpw Password for the user to bind to LDAP
+## DEPRECATED ldap.search_attr It will removed in a future, please use 'ldap.searchAttribute' instead
+## DEPRECATED ldap.search_filter It will removed in a future, please use 'ldap.searchFilter' instead
+## @param ldap.searchAttribute Attribute to match against the user name in the search
+## @param ldap.searchFilter The search filter to use when doing search+bind authentication
 ## @param ldap.scheme Set to `ldaps` to use LDAPS
 ## @param ldap.tls Set to `1` to use TLS encryption
 ##
 ldap:
   enabled: false
+  uri: ""
   server: ""
   port: ""
   prefix: ""
   suffix: ""
-  baseDN: ""
-  bindDN: ""
-  bind_password: ""
-  search_attr: ""
-  search_filter: ""
+  basedn: ""
+  binddn: ""
+  bindpw: ""
+  searchAttribute: ""
+  searchFilter: ""
   scheme: ""
   tls: ""
   ## @param ldap.url LDAP URL beginning in the form `ldap[s]://host[:port]/basedn`. If provided, all the other LDAP parameters will be ignored.
```

Comment:
Please move `ldap.uri` here. Since it is mutually exclusive with the others, I think visually separating them from the rest would help prevent users from accidentally setting both.